code/modules/hydroponics/grown/citrus.dm

```diff
@@ -132,24 +132,24 @@
 	qdel(src) //Ensuring that it's deleted by its own explosion
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/firelemon/proc/prime()
-	switch(seed.potency) //Combustible lemons are alot like IEDs, lots of flame, very little bang.
+	switch(seed.potency) //Explosion size is proportional to potency.
 		if(0 to 30)
 			update_mob()
 			explosion(src.loc,-1,-1,2, flame_range = 1)
 			qdel(src)
 		if(31 to 50)
 			update_mob()
-			explosion(src.loc,-1,-1,2, flame_range = 2)
+			explosion(src.loc,-1,-1,3, flame_range = 2)
 			qdel(src)
 		if(51 to 70)
 			update_mob()
-			explosion(src.loc,-1,-1,2, flame_range = 3)
+			explosion(src.loc,-1,-1,3, flame_range = 4)
 			qdel(src)
 		if(71 to 90)
 			update_mob()
-			explosion(src.loc,-1,-1,2, flame_range = 4)
+			explosion(src.loc,-1,1,3, flame_range = 5)
 			qdel(src)
 		else
```

Comment:
Just be aware that if somehow the seed's potency was less than 0 or greater than 90 then you get the biggest explosion, although I don't think you can get a negative seed potency